### DISCUSSION TITLE
Need dark theme support in editor

### DISCUSSION THREAD

**Discussion by yashvi2026** [GrapesJS/grapesjs#6196] [General]
It would be great if you add theme switching in the editor, allowing me to toggle between dark and light themes based on my preference.

**Comment by pruge** [comment]
Talk about the experience of using next, next-themes, tailwind.

When you register the following files in scripts in canvas config, theme is linked.

```js
// public folder, filename: theme.js
const doc = window.document
const theme = localStorage.getItem('theme')
doc.body.classList.remove('light', 'dark')
doc.body.classList.add(theme || '')
```

```ts
  const options: EditorConfig = {
    canvas: {
      scripts: ['/theme.js'],
    },
  }
```
